### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by olivergs** [eyeos/spice-web-client#9]
Hi there

After looking into files I've noticed that there are several files without headers or licenses of any type. This are the files I've found that need licensing information:

process/*
keymaps/*
lib/images/* (except png.js with LGPL)
lib/encrypt.js
lib/sha1.js
lib/prng4.js (http://www-cs-students.stanford.edu/~tjw/jsbn/LICENSE)
lib/rng.js (http://www-cs-students.stanford.edu/~tjw/jsbn/LICENSE)

Could you specify what license should I use if I want to redistribute this code?

**Comment by JJ** [comment]
Related #3 
It would better if they were moved out of the repo.

**Comment by aruiz** [comment]
Yes, the external ones are, but there's a bunch of files that are not clear whether they are external deps or headerless source of the spice client itself. We want clarification before we redistribute anything.

**Comment by JJ** [comment]
2016-02-17 12:31 GMT+01:00 Alberto Ruiz <notifications@github.com>:

> Yes, the external ones are, but there's a bunch of files that are not
> clear whether they are external deps or headerless source of the spice
> client itself. We want clarification before we redistribute anything.
>

And links to the C source, just in case you need to debug everything :-)

(private joke, just move along)

**Comment by Fantu** [comment]
About keymaps/keymapit.js I added the license is the same of the project (agplv3) but without any addition(hoping is right), I'll do a PR with header (at least for the keymap file I added).
I was waiting for a change from the eyeos developers before adding header with same license in the file but no reply for months :(
About the last license post done by developer:
https://lists.freedesktop.org/archives/spice-devel/2015-November/023313.html
Some days ago I tried to write also another post about the old and this discussion:
https://lists.freedesktop.org/archives/spice-devel/2016-March/027081.html
